client/src/app/site/users/components/user-list/user-list.component.ts

```diff
@@ -99,4 +101,18 @@ export class UserListComponent extends ListViewBaseComponent<ViewUser> implement
     public onPlusButton(): void {
         this.router.navigate(['./new'], { relativeTo: this.route });
     }
+
+    // TODO save all data from the dataSource
+    public saveUserList(): void {
+        this.csvExport.saveAsCsv(
+            this.dataSource.data,
+            [
```

Comment:
```
{
    fullName: 'Name',
    groups: 'Groups',
    isActive: 'Presence'
}
```

Reply:
(direct discussion: we keep it an array of objects: Order will be preserved, and label can be optional (capitalized version of the property will be translated and used, if no label present)